array/DNAm/analysis/methodsDevelopment/simulateCellSpecificEWAS.r

```diff
@@ -231,62 +234,64 @@ for(simNum in 1:nSim){
 			outtab.sim<-outtab.null
 			outtab.sim[sigProbes,]<-outtab.sig
 			
-			sumSim[rowNum,2+seq(1,6*9,6)]<-colSums(outtab.sim < 9e-8)
-			sumSim[rowNum,3+seq(1,6*9,6)]<-colSums(outtab.sim[sigProbes,] < 9e-8)
-			sumSim[rowNum,4+seq(1,6*9,6)]<-colSums(outtab.sim[-sigProbes,] < 9e-8)
-			
-			# substract those significant in both ME and Int from ME count

```

Comment:
What do ME and Int stand for?

Reply:
ME stands for main effect and Int interaction. They refer to the two terms in the linear model used to identify DMPs.